.ci/test_windows.ps1

```diff
@@ -6,6 +6,20 @@ function Check-Output {
   }
 }
 
+# unify environment variables for Azure devops and AppVeyor
+if (Test-Path env:APPVEYOR) {
+  $env:APPVEYOR = "true"
+  $env:BUILD_SOURCESDIRECTORY = $env:APPVEYOR_BUILD_FOLDER
+}
+
+# setup for Python
+conda init powershell
+conda activate
+conda config --set always_yes yes --set changeps1 no
+conda update -q -y conda
+conda create -q -y -n $env:CONDA_ENV python=$env:PYTHON_VERSION joblib matplotlib numpy pandas psutil pytest python-graphviz "scikit-learn<=0.21.3" scipy wheel ; Check-Output $?
```

Comment:
Why do we need `wheel` here explicitly?

Reply:
I was getting a failure with `bdist` because `bdist_wheel` wasn't recognized. But I know that task has been working for a while so maybe it was something else I was doing wrong. I'll try removing it